src/apps/Settings/DaoSettings.js

```diff
@@ -149,4 +151,13 @@ const ButtonLink = styled(Button).attrs({ mode: 'text' })`
   text-decoration: underline;
 `
 
-export default DaoSettings
+export default props => (
+  <React.Fragment>
+    <BreakPoint to="medium">
+      <DaoSettings {...props} shorten />
```

Comment:
Coming in _super_ late to this, but what do you think about changing this to `shortenAddresses` to be more descriptive?

Reply:
Sure, it is a small thing that can be done very quickly... [done](https://github.com/aragon/aragon/pull/580) :)